### PR TITLE
Adding Markdown link syntax for URL

### PR DESCRIPTION
**Pull request by JasonCard**
URL without using Markdown link syntax can cause broken hyperlink for localized languages, so adding the syntax to resolve the issue.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/machine-learning/tutorials/image-classification.md [hunk 1]
```diff
@@ -61,9 +61,9 @@ Image Classification is a common Machine Learning task that allows us to automat
 >[!Note]
 > The preceding images belong to Wikimedia Commons and are attributed as follows:
 >
-> * "220px-Pepperoni_pizza.jpg" Public Domain, https://commons.wikimedia.org/w/index.php?curid=79505,
-> * "119px-Nalle_-_a_small_brown_teddy_bear.jpg" By [Jonik](https://commons.wikimedia.org/wiki/User:Jonik) - Self-photographed, CC BY-SA 2.0, https://commons.wikimedia.org/w/index.php?curid=48166.
-> * "193px-Broodrooster.jpg" By [M.Minderhoud](https://nl.wikipedia.org/wiki/Gebruiker:Michiel1972) - Own work, CC BY-SA 3.0, https://commons.wikimedia.org/w/index.php?curid=27403
+> * "220px-Pepperoni_pizza.jpg" Public Domain, [https://commons.wikimedia.org/w/index.php?curid=79505](https://commons.wikimedia.org/w/index.php?curid=79505).
```
Comment (mairaw) [inline docs/machine-learning/tutorials/image-classification.md:64]:
```suggestion
> * "220px-Pepperoni_pizza.jpg" Public Domain, <https://commons.wikimedia.org/w/index.php?curid=79505>.
```

File: docs/machine-learning/tutorials/image-classification.md [hunk 2]
```diff
@@ -61,9 +61,9 @@ Image Classification is a common Machine Learning task that allows us to automat
...
+> * "119px-Nalle_-_a_small_brown_teddy_bear.jpg" By [Jonik](https://commons.wikimedia.org/wiki/User:Jonik) - Self-photographed, CC BY-SA 2.0, [https://commons.wikimedia.org/w/index.php?curid=48166](https://commons.wikimedia.org/w/index.php?curid=48166).
```
Comment (mairaw) [inline docs/machine-learning/tutorials/image-classification.md:65]:
```suggestion
> * "119px-Nalle_-_a_small_brown_teddy_bear.jpg" By [Jonik](https://commons.wikimedia.org/wiki/User:Jonik) - Self-photographed, CC BY-SA 2.0, <https://commons.wikimedia.org/w/index.php?curid=48166>.
```

File: docs/machine-learning/tutorials/image-classification.md [hunk 3]
```diff
@@ -61,9 +61,9 @@ Image Classification is a common Machine Learning task that allows us to automat
...
+> * "193px-Broodrooster.jpg" By [M.Minderhoud](https://nl.wikipedia.org/wiki/Gebruiker:Michiel1972) - Own work, CC BY-SA 3.0, [https://commons.wikimedia.org/w/index.php?curid=27403](https://commons.wikimedia.org/w/index.php?curid=27403).
```
Comment (mairaw) [inline docs/machine-learning/tutorials/image-classification.md:66]:
```suggestion
> * "193px-Broodrooster.jpg" By [M.Minderhoud](https://nl.wikipedia.org/wiki/Gebruiker:Michiel1972) - Own work, CC BY-SA 3.0, <https://commons.wikimedia.org/w/index.php?curid=27403>.
```

**Comment by JasonCard** [comment]
Closing as same fix is in [#18470](https://github.com/dotnet/docs/pull/18470).

**Comment by luisquintanilla** [comment]
Thanks @JasonCard and @mairaw . I knew I had seen these changes before, so I was confused. Thanks again for taking a look at it though!

**Pull request closed without merging**